olive/evaluator/evaluation.py

```diff
@@ -137,12 +145,12 @@ def compute_latency(latencies, sub_type) -> float:
     return latency_metrics[sub_type]
 
 
-def evaluate_custom_metric(model: OliveModel, metric: Metric, device: Device = Device.CPU):
+def evaluate_custom_metric(model: OliveModel, metric: Metric, device: Device = Device.CPU, execution_providers=None):
     _, _, eval_func = get_user_config(metric.user_config)
 
     if not eval_func:
         raise Exception("Please specify 'evaluate_func' for custom metrics")
-    return eval_func(model, metric.user_config.data_dir, metric.user_config.batch_size, device)
+    return eval_func(model, metric.user_config.data_dir, metric.user_config.batch_size, device, execution_providers)
```

Comment:
Can you also update the documentation here for custom metric func https://microsoft.github.io/Olive/tutorials/configure_metrics.html#custom-metric?